microservice/package/tmp/resources/entrypoint.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+if [ -n "$MEMORY_LIMIT" ];
```

Comment:
these are for junit tests, right? Can these files under tmp/ moved to microservice/package/test/resources instead?

Reply:
Removed and added to .gitignore, going to be gone with next commit